security/fma-supervisor.md

```diff
@@ -0,0 +1,215 @@
+    - Operators may not appreciate the increased burdeon and could decline to validate the network at all
+- Risk Assessment
+    - Sliding Scale Impact and Likelihood
+    - Network security is not based on number of validators, but fewer people running the OP Stack means less validation, mindshare, etc.
+- Mitigation
+    - There is a feature which we have not yet had a chance to implement called "Standard Mode", which is a counter to "Managed Mode"
+        - In Standard Mode, a Node *is not* managed by a Supervisor, and instead runs its on L1 derivation
+        - Periodically, the Node reaches out to *some* trusted Supervisor to fetch the Cross-Heads and any Replacement signals
+        - The Node uses this data to confirm that the chain it has derived is also cross-safe, and to handle invalid blocks
+    - With Standard Mode, operators would only *need* to run a single Node, in exchange for the interop aspects of validation being a trusted activity. We expect this Mode to be valuable to offset operator burdeon in cases where fully trustless validation isn't critical.
+
+## FM6: Standard Mode Nodes Trust a Malicious Supervisor
+
+- Description
+    - A Node is in Standard Mode, doing derivation on the L1 independently
+    - At some moment, it reaches out to a trusted Supervisor endpoint to query for Cross Safety and Replacement information
+    - The Supervisor is incorrect, or even adversarial
+    - The result depends on the way in which the data is incorrect:
+        - If the data ignores a block Invalidation/Replacement, the Node will be deceived into following a chain where an invalid interop message exists, allowing for invalid interop behaviors (like inappropriate minting) *on that Node* (and presumably all Nodes who also trust this Supervisor)
+        - If the data claims a block should be replaced, the Node would similarly perform the replacement, leaving this Node at that state.
+        - If the data simply isn't consistently applicable to what the Node independently derived, the Node has nothing to be decieved about, but also can't make forward progress, and would need to halt at this point.
+- Risk Assessment
+    - Low Impact, Unknown Likelihood
+    - If there were a tactical reason to use a Trusted Sequencer Endpoint to fool a Node as part of a larger exploit, this would be an attractive thing to attempt.
+    - Most entities who would provide a Trusted Endpoint have intrinsic incentive to keep their endpoint valid and secure. Much like a provider like Alchemy doesn't want the Node you connect to to lie.
+    - At most, a dishonest Supervisor can mislead the Nodes connected to it. So long as block producers *are not* using one of these Trusted Endpoints, block production is unaffected.
+- Mitigation
+    - Standard Mode could allow for *multiple* Supervisor Endpoints to be specified, they could confirm that all endpoints agree, preventing dishonesty from one party from deceiving the Node.
```

Comment:
Not viable now, but later with interop, we can potentially include proofs for trustless sync. Since all L2 data is really just some computation over L1 data, which the op-node can verify if they've access to a L1 header chain.